src/engine/sys/sdl_glimp.cpp

```diff
@@ -1428,6 +1429,23 @@ void GLimp_HandleCvars()
 
 		r_fullscreen->modified = false;
 	}
+
+	if ( r_noBorder->modified )
+	{
+		bool needToToggle = true;
```

Comment:
Drop this `needToToggle` check. I don't think it's important to avoid doing SetWindowBordered(true) twice if the user sets the cvar to 1 and then 2, and you already have the `modified` check to avoid setting it every frame.

Reply:
Oh, I missed this message…

Reply:
Fixed, merged.